charts/fpl-case-service/values.aat.template.yaml

```diff
@@ -13,6 +13,7 @@ java:
     SPRING_SECURITY_ENABLED: true
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI: https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI: https://idam-api.aat.platform.hmcts.net/o/jwks
+    DOCUMENT_MANAGEMENT_GATEWAY_URL: https://gateway-${SERVICE_FQDN}
```

Comment:
Not sure why you insist of using the document management, I am not sure if we should say it's document management, as this is just generic gateway? Different purpose?

Reply:
for now was trying to make it as specific as possible. But you have made a good

Reply:
Lets keep as gateway given the class name 👍